src/styles/components/header.scss

```diff
@@ -1,2 +1,91 @@
+@import '../mixins';
+
 .header {
+    background: var(--color-background-darker);
+    font-size: 1.6rem;
+    border-bottom: 0.1rem solid #5d5c61;
```

Comment:
add a variable for this  and make it `rgba(255,255,255,0.25)` instead of the hex value

Reply:
Done